test/disability-benefits/526EZ/config/vaFacilities.unit.spec.jsx

```diff
@@ -38,23 +38,23 @@ describe('Disability benefits 526EZ VA facility', () => {
         uiSchema={uiSchema}/>
     );
 
-    fillData(form, 'select#root_vaTreatments_0_treatmentDateRange_fromMonth', '1');
-    fillData(form, 'select#root_vaTreatments_0_treatmentDateRange_fromDay', '3');
-    fillData(form, 'input#root_vaTreatments_0_treatmentDateRange_fromYear', '1950');
-    fillData(form, 'select#root_vaTreatments_0_treatmentDateRange_toMonth', '1');
-    fillData(form, 'select#root_vaTreatments_0_treatmentDateRange_toDay', '3');
-    fillData(form, 'input#root_vaTreatments_0_treatmentDateRange_toYear', '1955');
-    fillData(form, 'input#root_vaTreatments_0_treatmentCenterName', 'Local facility');
+    fillData(form, 'select#root_treatments_0_treatmentDateRange_fromMonth', '1');
```

Comment:
For what it's worth, I had trouble when I tried switching these to `fillDate` and I didn't have the time to investigate. Wondering if you ran into issues as well or if you just decided to leave it as-is for the sake of keeping the PR focused?

Reply:
Yeah, I didn't try. Suppose I could give it a whirl.